sunpy/coordinates/utils.py

```diff
@@ -436,3 +437,36 @@ def solar_angle_equivalency(observer):
               lambda x: np.arctan(x/sun_observer_distance))]
 
     return equiv
+
+
+@u.quantity_input
```

Comment:
This `quantity_input` decorator isn't doing anything.  Presumably you intended to add a unit annotation for the `rsun` argument?

Reply:
Thanks - I either missed this or assumed that setting a default keyword value to a quantity was sufficient. Should be fixed now.